asserts/database.go

```diff
@@ -311,9 +338,12 @@ func (db *Database) FindMany(assertionType *AssertionType, headers map[string]st
 	foundCb := func(assert Assertion) {
 		res = append(res, assert)
 	}
-	err = db.bs.Search(assertionType, headers, foundCb)
-	if err != nil {
-		return nil, err
+
+	for _, bs := range db.backstores {
+		err = bs.Search(assertionType, headers, foundCb)
```

Comment:
Is there any way we can get into the situation where a trusted key already exists in the non-trusted backstore? If so, that key would be returned to the caller twice.

The trusted backstore should be tiny so maybe it makes sense to record trusted matches as they're found and filter them from the non-trusted results. But ... that means knowing which backstore is trusted which would make this code a bit uglier :/.